tests/admin_inlines/tests.py

```diff
@@ -906,3 +906,42 @@ def test_collapsed_inlines(self):
             self.wait_until_visible(field_name)
             hide_links[hide_index].click()
             self.wait_until_invisible(field_name)
+
+    def test_non_autofield_primary_key(self):
+        """
+        Test for #15665 where inline wouldn't work if using non-AutoField ids and inherinting the model
```

Comment:
State the expected behavior rather than "Test ...." (all tests test things).
No need for a selenium test as no JavaScript is involved. See 3aad955ea8db1592fad0012155eaa25b72e50dc5 for similar changes.